### DISCUSSION TITLE
CommandBase example

### DISCUSSION THREAD

**Discussion by oskarjf** [MarimerLLC/csla#3272] [Questions] [answered]
I'm upgrading to 5.5.3 but none of the samples in the repo show the usage of CommandBase. Where can I see an example of the usage?

/Oskar

**Comment by danielmartind** [comment, marked as answer]
There is an example in  [Project Tracker](https://github.com/MarimerLLC/csla/blob/main/Samples/ProjectTracker/ProjectTracker.BusinessLibrary.Netstandard/ProjectExistsCommand.cs)
